src/proto/lbann.proto

```diff
@@ -739,6 +746,8 @@ message Layer {
    StopGradient stop_gradient = 319;
    Max max = 320;
    Min min = 321;
+   BinaryCrossEntropyLoss bce = 322;
```

Comment:
I think it would be much clearer if we named these "binary_cross_entropy" and ""mean_squared_error" instead of using initials.